.github/workflows/publish-on-tag.yml

```diff
@@ -37,3 +37,15 @@ jobs:
           npm config set registry 'https://wombat-dressing-room.appspot.com/'
           npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
           npm publish
+
+  post-publish:
+    needs: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Bump to Post Version
+        run: |
+          VERSION=$(node utils/apply_post_version.js)
+          npm run doc
```

Comment:
the command sometimes says that it requires `npm run tsc`, should we re-compile after we got applied the version?